shared/templates/template_ANSIBLE_audit_rules_dac_modification

```diff
@@ -37,7 +37,7 @@
 - name: Inserts/replaces the {{{ ATTR }}} rule in rules.d when on x86
   lineinfile:
     path: "{{ all_files[0] }}"
-    line: "-a always,exit -F arch=b32 -S {{{ ATTR }}} -F auid>=1000 -F auid!=4294967295 -F key=perm_mod"
```

Comment:
Can the `auid` in Ansible tasks be changed to `unset` as well?

Reply:
Nice catch, fixed!
Also fixed some other places I forgot about.